src/cryptoadvance/specterext/notifications/app_config.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
Also the app_config doesn't make sense for internal extensions. Please delete this file.

Reply:
Done https://github.com/cryptoadvance/specter-desktop/pull/1766/commits/2de3e1e20677fe1578d315e77e952c41304adf5d